doc/lsp.txt

```diff
@@ -331,6 +331,21 @@ accepts a list of LSP servers with the above information.
 :LspHover		Show the documentation for the symbol under the cursor
 			in a popup window.
 
+			Var 'b:lsphoverpopupid' and 'b:lsphovermarkupkind' as
+			hover popup id and its markup kind ("plaintext" or
+			"markdown" or "") added when hover popup showed.
```

Comment:
When the hover popup contains markdown text, the markdown characters also will be displayed in the buffer. These characters need to be processed and hidden. So it is not sufficient to just highlight the characters in the hover popup. The hover popup buffer contents needs to be updated based on the markdown syntax.  I am not yet sure how this will work. Instead of expecting another plugin or the user to do this, it will be useful if the LSP plugin itself handles this.

Reply:
if it's a concern, actually can just get rid of first-line '```' and last-line '```' (if it is markdown markup kind), then treat it as plaintext at all.
// but i prefer one day someone would  dev a common syntax for 'lsphover' ft, just like others in vim runtime/syntax.

Reply:
i mean here now i/we just offerred an interface to user, then he/she can customize something as he/she preferred;
as for you said if the LSP plugin itself handled this, i think it's good if can, but now, perhaps we can just give interface as first step.